### PR TITLE
[bitnami/vault] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/vault/templates/injector/service.yaml [hunk 1]
```diff
@@ -57,7 +49,8 @@ spec:
     {{- if .Values.injector.service.extraPorts }}
     {{- include "common.tplvalues.render" (dict "value" .Values.injector.service.extraPorts "context" $) | nindent 4 }}
     {{- end }}
-  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
+  {{- $podLabels := merge .Values.injector.podLabels .Values.commonLabels }}
```
Comment (javsalgar) [inline bitnami/vault/templates/injector/service.yaml:52]: Performed a manual check and confirmed that the `selector` field in a service is mutable, so there should be no issues during upgrades.

File: bitnami/vault/templates/server/service.yaml [hunk 2]
```diff
@@ -9,23 +9,12 @@ kind: Service
...
-    {{- if .Values.server.service.general.annotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.server.service.general.annotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if and .Values.server.metrics.enabled .Values.server.metrics.annotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.server.metrics.annotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-    {{- end }}
+  {{- if or .Values.server.service.annotations .Values.commonAnnotations }}
```
Comment (javsalgar) [inline bitnami/vault/templates/server/service.yaml:15]: It should be `server.service.general.annotations`

**Review by javsalgar — CHANGES_REQUESTED** [review]
Nice! Small change requested

File: bitnami/vault/templates/server/service.yaml (hunk 2 above)
Comment (juan131) [reply bitnami/vault/templates/server/service.yaml]: fixed at https://github.com/bitnami/charts/pull/18445/commits/ad2828baf71e20c61c08b33cc2ed9534a05c7b9e

**Pull request merged**